Source/ACE/Network/GameEvent/Events/GameEventPlayerDescription.cs

```diff
@@ -157,7 +160,7 @@ private void WriteEventBody()
                 }
             }
 
-            var propertiesDid = aceObj.DataIdProperties.Where(x => x.PropertyId < 9000).ToList();
+            var propertiesDid = aceObj.DataIdProperties.Where(x => x.PropertyId < 9000 && x.PropertyValue != null).ToList();
```

Comment:
Do we want to allow null properties into our AceObject and/or database?  This function seems to assume that they will propagate there.